tests/stencils/test_vertical_index_fields.py

```diff
@@ -0,0 +1,266 @@
+from dusk.script import *
+from test_util import transpile, validate
+
+
+def test_vertical_index_fields():
+    validate(transpile(simple_example))
+    # FIXME: add validation again
+    transpile(various_expression)
+    transpile(index_fields_with_offsets)
+    transpile(various_dimensions_mix)
+    transpile(sparse_index_fields)
+
```

Comment:
why can't we validate these examples? do all of them trigger a versioning of a sparse field? if its because of the catalogued neighbor chains, we could still validate them using the cxx naive backend, no?

Reply:
`validate` doesn't run either of the backends, but only the validation passes _accessible_ in dawn4py. Validation failed when I tried it. I'm not sure what the problem is, but I have the feeling that a few of the recent dawn fixes could fix the problem here.